compiler/src/iree/compiler/Codegen/Common/test/fission_transfer_ops_control_flow.mlir

```diff
@@ -1,11 +1,12 @@
-// RUN: iree-opt --split-input-file -pass-pipeline="builtin.module(func.func(iree-codegen-fission-transfer-ops-in-control-flow),cse,canonicalize)" %s | FileCheck %s
+// RUN: iree-opt --split-input-file -pass-pipeline="builtin.module(func.func(iree-codegen-fission-transfer-ops-in-control-flow{fission-multi-trip}),cse,canonicalize)" %s | FileCheck %s --check-prefixes=CHECK-ALL,MULTI
+// RUN: iree-opt --split-input-file -pass-pipeline="builtin.module(func.func(iree-codegen-fission-transfer-ops-in-control-flow),cse,canonicalize)" %s | FileCheck %s --check-prefixes=CHECK-ALL,SINGLE
 
-// CHECK-LABEL: @fission_global_read_to_private_write
-// CHECK-SAME: %[[ARG0:.*]]: memref<1x?x?x8xbf16, #amdgpu.address_space<fat_raw_buffer>>
-// CHECK-SAME: %[[ARG1:.*]]: index
-// CHECK-SAME: %[[ARG2:.*]]: i1
-// CHECK-SAME: %[[ARG3:.*]]: vector<1x1x1x8xbf16>
-// CHECK-SAME: %[[ARG4:.*]]: memref<1x1x1x8xbf16, #gpu.address_space<private>>
+// CHECK-ALL-LABEL: @fission_global_read_to_private_write
+// MULTI-SAME: %[[ARG0:.*]]: memref<1x?x?x8xbf16, #amdgpu.address_space<fat_raw_buffer>>
+// MULTI-SAME: %[[ARG1:.*]]: index
+// MULTI-SAME: %[[ARG2:.*]]: i1
+// MULTI-SAME: %[[ARG3:.*]]: vector<1x1x1x8xbf16>
+// MULTI-SAME: %[[ARG4:.*]]: memref<1x1x1x8xbf16, #gpu.address_space<private>>
```

Comment:
Nit: Probably doesn't make a difference today since those `SINGLE` test statements doesn't involve any variables and they don't need to be.

I think `CHECK-ALL-SAME:` is better than `MULTI-SAME:` because it will give remaining unit test statements to have access to function arguments. This allows us to extend the `SINGLE` test statements easier should we need to in the future.

This applies to multiple test cases below.